t/40-expand_placeholders.t

```diff
@@ -0,0 +1,124 @@
+#! /usr/bin/perl
+
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+use strict;
+use warnings;
+
+BEGIN {
+    unshift @INC, 'lib';
+}
```

Comment:
I know this BEGIN block is used in many other tests.
Just a note that this was the way to do it in older perls, but the modern way is to use
```perl
use FindBin;
use lib "$FindBin::Bin/../lib";
```